app/parse.py

```diff
@@ -8,8 +13,52 @@ class Quote:
     tags: list[str]
 
 
+QUOTES_FIELDS = [fields.name for fields in fields(Quote)]
```

Comment:
The list comprehension uses `fields` as both the module name and the loop variable, which can cause confusion. Consider renaming the loop variable to avoid shadowing the imported `fields` function.